netsim/ansible/templates/initial/cumulus_nvue.j2

```diff
@@ -1,3 +1,40 @@
+{% macro decl_interface(i) %}
+      {{ i.ifname }}:
+        link:
+{%  if i.mtu is defined %}
+          mtu: {{ i.mtu }}
+{%  endif %}
+          state:
+            up: {}
+{%  if i.name is defined %}
+        description: "{{ i.name }}{{ " ["+i.role+"]" if i.role is defined else "" }}"
+{%  elif i.type|default("") == "stub" %}
+        description: Stub interface
+{%  endif %}
+{%  if i.ipv4 is defined or i.ipv6 is defined %}
+        ip:
+          address:
+{%   if i.ipv4 is defined %}
+{%    if i.ipv4 == True %}
+            {{ loopback.ipv4 }}: {}
+{%    else %}
+            {{ i.ipv4 }}: {}
+{%    endif %}
+{%   endif %}
+{%     if i.vrf is defined %}
```

Comment:
I prefer having a 2-space indents (makes things easier to read, at least for me), but whatever you do, please be consistent.

Reply:
Would it help to use something like https://www.djlint.com/docs/configuration/?

Reply:
Probably. Thank you, will check it out.